e2etest/get-testcases.py

```diff
@@ -40,6 +40,7 @@
     perf_matrix = {"testcase": [], "testing_ami": ["soaking_linux"], "data_rate": ["100", "1000", "5000"]}
     canary_matrix = {"testcase": [], "testing_ami": ["canary_linux", "canary_windows"]}
     containerinsight_eks_prometheus_matrix = {"testcase": []}
+    containerinsight_eks_matrix = {"testcase": []}
```

Comment:
Can we merge container insight test cases into the same matrix? Each matrix will create an independent test set, and I feel prometheus and infra should be in the same. 
For example
https://github.com/aws-observability/aws-otel-collector/actions/runs/757459808

Reply:
Sure. I think they should be merged.